packages/pigeon/lib/java_generator.dart

```diff
@@ -436,14 +454,15 @@ class JavaGenerator extends StructuredGenerator<JavaOptions> {
               indent.writeln('@SuppressWarnings("ConstantConditions")');
               if (func.returnType.baseName == 'int') {
                 indent.writeln(
-                    '$returnType $output = channelReply == null ? null : ((Number)channelReply).longValue();');
+                    '$returnType $output = channelReply == null ? null : ((Number) channelReply).longValue();');
               } else {
                 indent.writeln(
-                    '$returnType $output = ($returnType)channelReply;');
+                    '$returnType $output = ($returnType) channelReply;');
```

Comment:
Optional: it might be worth extracting a `getCastExpression({required String type, required String value})` to abstract this since we've messed up the format in several places. Then the exact format of a cast only needs to be right in one place.